### PR TITLE
Psr 437 438

### PR DESCRIPTION
**Pull request by EricSiegNW**
Subclass Instruction text, center the title text, adjust image height for the instruction steps to always be the same.

### CODE REVIEW

File: Psorcast/Psorcast/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -300,7 +300,7 @@ OCT 2020 to Today</string>
                                                                                 </constraints>
                                                                             </view>
                                                                             <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="CxB-DG-SQx" customClass="RSDRoundedButton" customModule="ResearchUI">
-                                                                                <rect key="frame" x="32" y="270" width="317" height="30"/>
+                                                                                <rect key="frame" x="32" y="248" width="317" height="52"/>
```
Comment (dephillipsmichael) [inline Psorcast/Psorcast/Base.lproj/Main.storyboard:303]: This doesn't seem related to this PR, is it?

**Review by dephillipsmichael — CHANGES_REQUESTED** [review]
I'm not sure we want to point SageResearch at your fork's branch.  So holding off on approving this.

An easier route would be to just make your own xib for the OnboardingInstructionViewController, because even if you merge into SageResearch, you will have to back-version the spot and have Carthage and all the other sub-modules now point at that new version.

**Comment by EricSiegNW** [comment]
> I'm not sure we want to point SageResearch at your fork's branch. So holding off on approving this.
> 
> An easier route would be to just make your own xib for the OnboardingInstructionViewController, because even if you merge into SageResearch, you will have to back-version the spot and have Carthage and all the other sub-modules now point at that new version.

Why can't we just merge into SageResearch and point it at the latest? The only changes SageResearch since our old head are a 2 line change by Shannon related to a voicebox thing which seems fine to pull in and you making a similar change a month ago to expose an internal function in the RSDStepViewController.

Seems like we can merge into SageResearch, point at latest, the end, rather than duplicating an existing class and Xib.

**Comment by EricSiegNW** [comment]
There's also definitely a weird situation where our project points at an almost modern version of SageResearch but MotorControl appears to point at a 2019 version.

**Comment by dephillipsmichael** [comment]
> > I'm not sure we want to point SageResearch at your fork's branch. So holding off on approving this.
> > An easier route would be to just make your own xib for the OnboardingInstructionViewController, because even if you merge into SageResearch, you will have to back-version the spot and have Carthage and all the other sub-modules now point at that new version.
> 
> Why can't we just merge into SageResearch and point it at the latest? The only changes SageResearch since our old head are a 2 line change by Shannon related to a voicebox thing which seems fine to pull in and you making a similar change a month ago to expose an internal function in the RSDStepViewController.
> 
> Seems like we can merge into SageResearch, point at latest, the end, rather than duplicating an existing class and Xib.

You're most likely right that it can be done.  I'm jaded from all my time wasted on messing with that stuff, go for it.  I'll remove my review.

**Comment by DanWebster** [comment]
@EricSiegNW Is this ready to merge now, or should we wait on pending Sage Research changes?

**Review by DanWebster — APPROVED** [review]

**Pull request merged**